packages/altair-app/src/app/modules/altair/services/pre-request/pre-request.service.ts

```diff
@@ -192,12 +202,22 @@ export class PreRequestService {
       },
       importModule: (moduleName: string) => this.importModuleHelper(moduleName),
       response: this.buildContextResponse(data),
+      log: (d: any) => {
```

Comment:
@imolorhe just curious, but since this `log` override is in `pre-request.service.ts`, does a similar change need to be make to `post-request.service.ts`?

The post-request tab is where personally I'm doing "post process & console.log".

That said, I'm just naively scanning the PR, so if the post-request tab ends up getting this functionality as well, then nevermind/np! Thanks!

Reply:
There's a preview link in the PR. You can always test the change there 🙂 However, we don't have a `post-request.service.ts`. We just have the `pre-request.service.ts` which powers both. I just haven't renamed it since post requests were introduced.

Reply:
Nice! Just tested it and it looks great!